resources.go

```diff
@@ -162,6 +167,20 @@ func getStringOrStringFromFile(field json.RawMessage) string {
 	return ""
 }
 
+func getBool(field json.RawMessage) bool {
+	var rawValue interface{}
+	if err := json.Unmarshal(field, &rawValue); err == nil {
+		switch rawValue := rawValue.(type) {
+		case bool:
+			return rawValue
+		default:
+			log.Printf("Could not parse bool from json; defaulting to false")
+			return false
```

Comment:
nit: maybe add the default value as a parameter for the function, to give some more control. Call it `getBoolOrDefault()` for example.

Reply:
@KevinSnyderCodes done: https://github.com/KevinSnyderCodes/github-deployment-resource/pull/9/files#diff-3d6b5e2ce37c7b5b2214e0a7d75ec1d7R170